docs/reference/grpc.md

```diff
@@ -0,0 +1,19 @@
+---
+title: 'Gadget gRPC API'
+weight: 30
+description: 'Reference documentation for the gRPC API'
+---
+
+TODO
+
+## api.proto
+
+[pkg/gadget-service/api/api.proto](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/pkg/gadget-service/api/api.proto)
```

Comment:
I think it does not work with Hugo once it is rendered in https://inspektor-gadget.io/docs

We currently have broken links on https://inspektor-gadget.io/docs/v0.22.0/architecture/ ("[Tracers](https://inspektor-gadget.io/docs/v0.22.0/pkg/gadgets/) : They are in charge of collecting events from the host, like process creation, file access, etc.")

Reply:
Opened https://github.com/inspektor-gadget/inspektor-gadget/issues/2254 as I think we need to fix this. It could create a lot of confusion to users if they are pointed to the wrong version of a file.